backend/src/authentication/dto/register-user.dto.ts

```diff
@@ -1,4 +1,18 @@
-import { IsEmail, IsNotEmpty, IsString, MinLength } from 'class-validator';
+import {
+  IsArray,
+  IsEmail,
+  IsEnum,
+  IsNotEmpty,
+  IsOptional,
+  IsString,
+  MinLength,
+} from 'class-validator';
+
+export enum RoleEnum {
+  ADMIN = 'ADMIN',
+  STAFF = 'STAFF',
+  ORGANIZER = 'ORGANIZER',
+}
```

Comment:
unfortunately, enums defined in prisma does not generate enums in prisma/client so i had to create it manually here